checkov/secrets/plugins/entropy_keyword_combinator.py

```diff
@@ -269,6 +275,14 @@ def analyze_line(
 
         return set()
 
+    def postprocess_secrets(self, detected_secrets: set[PotentialSecret], line: str) -> None:
+        for detected_secret in detected_secrets:
+            if detected_secret.secret_value and line.replace('"', '').replace("'", '').startswith(
```

Comment:
@maxamel Do we need to clean up more characters? whitespace?

Reply:
No, since if you remove whitespace from line you'd have to do it for secret value as well.